include/usvfs.h

```diff
@@ -49,6 +49,7 @@ static const unsigned int LINKFLAG_CREATETARGET   = 0x00000004; // if set, file
                                                                 // If there different create-target have been set for an element and one of its
                                                                 // ancestors, the inner-most create-target is used
 static const unsigned int LINKFLAG_RECURSIVE      = 0x00000008; // if set, directories are linked recursively
+static const unsigned int LINKFLAG_FAILIFSKIPPED  = 0x00000010; // if set, linking fails if the file or directory is skipped
```

Comment:
> Seems good, I reviewed from phone though. Have you been able to test and verify things are being skipped correctly?

Yes, it's been fairly well test-covered through https://github.com/ModOrganizer2/modorganizer/pull/2033

> Comment should mention the skip lists to give contetext as to how a file might actually be skipped. Also put the comment on a separate line above, simply to avoid long lines.

Updated